### ISSUE TITLE
Unable to search as "Terminal" when It is display as "终端"

### ISSUE THREAD

**Issue by youngzyl** [microsoft/terminal#12829]
Title: Unable to search as "Terminal" when Terminal is display as "终端"
Version: 1.12.10733.0
Windows version: Windows 11 workstation 21H2 22000.527
Description:

Once the Terminal upgrade the Chinese language, the name switch to "终端". Then you'll be not able to search it as "Terminal".

![image](https://user-images.githubusercontent.com/35056356/161735599-7a79057b-8e98-4f13-993f-eea11338c3da.png)
![image](https://user-images.githubusercontent.com/35056356/161735626-1a5ff5b1-22c4-41aa-95c0-8843021b3a75.png)

For modern UWP application, it usually can be search as English even if it has alternative traslation. the searches of Mail exemplify this situation.

![image](https://user-images.githubusercontent.com/35056356/161735952-9311f01f-350c-40a9-bef2-015d9b8b15da.png)

**Comment by zadjii-msft** [comment]
Huh. Good catch. I wonder if there's a way to have other search terms in our manifest that _aren't_ translated.

**Comment by skyline75489** [comment]
Yeah, this is actually also bothering me. It's much easier to search with "terminal" instead of "终端"

**Comment by DHowett** [comment]
We're working with the Start Menu search team to make sure that we're set up properly here. Thanks so much for the feedback!

**Comment by DHowett** [comment]
Unfortunately, we can't add aliases to the app manifest. It's gotta be pushed out by the search service. Yup.

**Comment by youngzyl** [comment]
@DHowett 
Thank you for your update.

I've received one working sample which is running at windows 11 22000.593, terminal version: 1.12.10733.0.
![image](https://user-images.githubusercontent.com/35056356/162579670-07d569d1-0b53-48c0-b132-18c872a7b4f3.png)
But I cannot reproduce on my machine.
![image](https://user-images.githubusercontent.com/35056356/162579714-c985e8cc-390e-40c7-8d94-bc3597313ce9.png)

**Comment by heruji** [comment]
Since 1.12, I always search "wt" to launch Terminal. But "wt" does not have "Run as admin" option in the search result.

(Another workaround is PowerToys Run, which has a Window Terminal plugin.)

Also, If both stable version and preview version installed, "wt" launches the stable version.
It seems the preview version does not have its own "wt" ?

![image](https://user-images.githubusercontent.com/5276960/163086968-6cde2eec-d085-4d8b-81da-14a03602eb45.png)

**Comment by zadjii-msft** [comment]
> Also, If both stable version and preview version installed, "wt" launches the stable version.
> It seems the preview version does not have its own "wt" in the search result ?

You change that under "app execution aliases" in the settings app:

![image](https://user-images.githubusercontent.com/18356694/163173681-e9e9038d-c714-44fc-9e1d-eb114e9924c5.png)

> And it would be nice if the search result has profiles listed as options.

It will for "Terminal", but not for wt.exe. See https://github.com/microsoft/terminal/issues/2339#issuecomment-776047929

**Comment by ershiwo** [comment]
In Powertoys Run, I also can't launch Terminal by the name "terminal" on a Win10 PC with Chinese language pack.

**Comment by youngzyl** [comment]
As a work around, you may search "teminal" instead, which is a little bit rediculous.

![image](https://user-images.githubusercontent.com/35056356/163662322-019efa2f-59b9-4046-be35-d817a0eab9b5.png)

**Comment by garmede** [comment]
The same thing happens in Korean.
![image](https://user-images.githubusercontent.com/2133201/166130827-53468888-cd8c-4fcc-84be-6b1dca9735c0.png)

**Comment by orange-pig** [comment]
Still exists in 22h2.
![image](https://user-images.githubusercontent.com/5195198/204178785-4d6ce944-f222-4d2d-a44e-a9cf88862f5c.png)
![image](https://user-images.githubusercontent.com/5195198/204178948-25799e02-5819-4acf-9d91-08724f264077.png)

**Comment by youngzyl** [comment]
Same for me, Issue can be reproed in the lastest published version of Windows 11 22H2
![image](https://user-images.githubusercontent.com/35056356/204179413-6519d482-7860-401b-ae99-ecbf2f646d9d.png)
![image](https://user-images.githubusercontent.com/35056356/204179502-528761c2-8d74-46dd-ba5c-4d717400ff8c.png)

**Comment by youngzyl** [comment]
![image](https://user-images.githubusercontent.com/35056356/222431753-b8b030b8-1958-485c-bb2d-7c8e2ce4e940.png)
Issue was reproduced in 22H2 Release Preview build 22621.1344

**Comment by kagura114** [comment]
This bug occurred after upgrading to the latest version (v1.17.11461.0) via MS store even if the system language is English, and reinstall with appbundle does no fix
![image](https://github.com/microsoft/terminal/assets/31371264/186f078e-519a-4384-a44d-f5d910703678)
![image](https://github.com/microsoft/terminal/assets/31371264/e766f519-8cbc-492c-b054-18e2693512a6)
![image](https://github.com/microsoft/terminal/assets/31371264/c58a7144-c884-465f-ac9b-c78e569ff286)

**Comment by FrzMtrsprt** [comment]
Bump. This issue is still present.

**Comment by yhmtsai** [comment]
I also face the same issue. My workaround is to add a shortcut into somewhere such as desktop and then pin it to startmenu if necessary.
The shortcut can be search by windows search or powertoy run.
1. right click on desktop -> add -> shortcut
target: `wt`
2. just rename the shortcut to Terminal
3. change the icon by terminal.ico from https://github.com/microsoft/terminal/tree/main/res
4. (If necessary), right click on shortcut to pin it in menu
![search](https://github.com/microsoft/terminal/assets/19565938/fc2b3115-8a14-41dc-b062-f89634c09022)
![powertoy](https://github.com/microsoft/terminal/assets/19565938/49eeb117-cafc-42e8-a88d-7531f87309b6)
